arch/x86/soc/intel_quark/quark_se/Kconfig.defconfig.series

```diff
@@ -17,13 +17,15 @@ config TOOLCHAIN_VARIANT
 	default "" if !X86_IAMCU
 
 config PHYS_RAM_ADDR
-	default 0xA8006400
+	default 0xA8007000 if X86_MMU
```

Comment:
We can continue to use that old value for situations which don't need page alignment. RAM space is never enough. So I think this is better to have this switch needed for MMU.
And on the ARC subsystem, since all my mmu code will be on x86 I don't have anything to modify correct.